assets/src/edit-story/components/panels/layer/layerList.js

```diff
@@ -18,60 +18,69 @@
  * External dependencies
  */
 import styled from 'styled-components';
-import { Fragment, useContext, useEffect } from 'react';
-
-/**
- * WordPress dependencies
- */
-import { sprintf, __ } from '@wordpress/i18n';
+import { Fragment, useContext } from 'react';
 
 /**
  * Internal dependencies
  */
-import useLiveRegion from '../../../utils/useLiveRegion';
+import {
+  Reorderable,
+  ReorderableSeparator,
+  ReorderableItem,
+} from '../../reorderable';
+import { useStory } from '../../../app';
 import Layer from './layer';
+import { LAYER_HEIGHT } from './constants';
 import LayerContext from './context';
-import LayerSeparator from './separator';
 
-const REORDER_MESSAGE = __(
-  /* translators: d: new layer position. */
-  'Reordering layers. Press Escape to abort. Release mouse to drop in position %d.',
-  'web-stories'
-);
-
-const LayerList = styled.div.attrs({ role: 'listbox' })`
-  display: flex;
+const LayerList = styled(Reorderable)`
```

Comment:
Might want to put `.attrs({'aria-orientation': 'vertical'})` on here too, though [it's default](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role).